bikeshed/test.py

```diff
@@ -12,13 +12,19 @@
 from .messages import *
 
 
+def findTestFiles():
+    for root, dirnames, filenames in os.walk(config.scriptPath() + "/../tests"):
+        for filename in filenames:
+            if filename.endswith(".bs"):
+                yield root + "/" + filename
```

Comment:
Mind using os.path.join() here?

And while we're here, swap out the os.walk argument to use the path functions too.